app/wasm/wasm.go

```diff
@@ -0,0 +1,18 @@
+package wasm
+
+import (
+	"github.com/CosmWasm/wasmd/x/wasm"
+	wasmkeeper "github.com/CosmWasm/wasmd/x/wasm/keeper"
+)
+
+func SetupWasmHandlers(
+	osmoKeeper ViewKeeper,
+) []wasmkeeper.Option {
+	queryPluginOpt := wasmkeeper.WithQueryPlugins(&wasmkeeper.QueryPlugins{
```

Comment:
Okay, I was going to suggest a refactor, but then I realized you will register custom message handlers here as well, so you need the array.

I would just rename to `RegisterCustomPlugins` or something